bin/hipcc

```diff
@@ -107,24 +107,20 @@ $HIP_RUNTIME= $hipConfig{'HIP_RUNTIME'};
 
 # If using VDI runtime, need to find HIP_VDI_HOME
 if ($HIP_RUNTIME eq "VDI" and !defined $HIP_VDI_HOME) {
-    $HIP_VDI_HOME = "/opt/rocm/hip"
+    $HIP_VDI_HOME = abs_path(dirname($0) . "/..");
```

Comment:
FYI: We have a symlink /opt/rocm/bin/hipcc that points to /opt/rocm/hip/bin/hipcc. Some users prefer to add /opt/rocm/bin to their PATH and directly execute hipcc. In which case HIP_VDI_HOME would point to /opt/rocm. This should be fine as long as the rest of the dependencies referred to by HIP_VDI_HOME are also symlinked correctly into /opt/rocm/bin, /opt/rocm/include and /opt/rocm/lib.

Reply:
fixed